octobot_trading/personal_data/portfolios/portfolio_manager.py

```diff
@@ -282,6 +282,10 @@ def _load_simulated_portfolio_from_history(self):
                 for symbol, value in self.historical_portfolio_value_manager.historical_ending_portfolio.items()
             }
         )
+        for asset_dict in portfolio_amount_dict.values():
```

Comment:
Should we add a test to ensure we don't break it again?

Reply:
good idea, i will

Reply:
done